autoPlay.js

```diff
@@ -98,37 +97,34 @@ function goToLaneWithBestTarget() {
 	// We can overlook spawners if all spawners are 40% hp or higher and a creep is under 10% hp
 	var spawnerOKThreshold = 0.4;
 	var creepSnagThreshold = 0.1;
-	
 	var targetFound = false;
 	var lowHP = 0;
 	var lowLane = 0;
 	var lowTarget = 0;
 	var lowPercentageHP = 0;
-	
 	// determine which lane and enemy is the optimal target
 	var enemyTypePriority = [
-		ENEMY_TYPE.TREASURE, 
-		ENEMY_TYPE.BOSS, 
+		ENEMY_TYPE.TREASURE,
+		ENEMY_TYPE.BOSS,
 		ENEMY_TYPE.MINIBOSS,
-		ENEMY_TYPE.SPAWNER, 
+		ENEMY_TYPE.SPAWNER,
 		ENEMY_TYPE.CREEP
 	];
-		
 	var skippingSpawner = false;
 	var skippedSpawnerLane = 0;
 	var skippedSpawnerTarget = 0;
 	var targetIsTreasureOrBoss = false;
-	
```

Comment:
Please leave whitespace changes from your code formatter out of the pull request - they'll cause unnecessary merge conflicts later otherwise.